test/functional/tests/fault_injection/test_fault_injection_many_to_one.py

```diff
@@ -0,0 +1,255 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+
+from api.cas import casadm, casadm_parser, cli
+from api.cas.cache_config import CacheMode, SeqCutOffPolicy
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_tools.dd import Dd
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+
+block_size = Size(1, Unit.Blocks4096)
+blocks_100MiB = int(Size(100, Unit.MebiByte).value / block_size.value)
+
+
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_remove_multilevel_core():
+    """
+        title: Test of the ability to remove a core used in a multilevel cache.
+        description: |
+          Negative test if OpenCAS does not allow to remove a core when the related exported object
+          is used as a core device for another cache instance.
+        pass_criteria:
+          - No system crash.
+          - OpenCAS does not allow removing a core used in a multilevel cache instance.
+    """
+    with TestRun.step("Prepare two devices for cache and one for core."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(1, Unit.GibiByte), Size(1, Unit.GibiByte)])
+        cache_part1 = cache_dev.partitions[0]
+        cache_part2 = cache_dev.partitions[1]
+        core_dev = TestRun.disks['core']
+        core_dev.create_partitions([Size(2, Unit.GibiByte)])
+        core_dev = core_dev.partitions[0]
+
+    with TestRun.step("Start the first cache instance"):
+        cache1 = casadm.start_cache(cache_part1, CacheMode.WT, force=True)
```

Comment:
WT is a default cache mode and you don't have to pass it as a parameter here and few lines below (line 48)